internal/core/command/main.go

```diff
@@ -73,8 +74,26 @@ func Main(ctx context.Context, cancel context.CancelFunc, router *mux.Router) {
 			NewBootstrap(router, common.CoreCommandServiceKey).BootstrapHandler,
 			telemetry.BootstrapHandler,
 			httpServer.BootstrapHandler,
-			handlers.NewStartMessage(common.CoreCommandServiceKey, edgex.Version).BootstrapHandler,
 		})
 
-	// code here!
+	if !success {
+		return
+	}
+
+	configuration = container.ConfigurationFrom(dic.Get)
+	if !handlers.NewMessagingBootstrap(configuration.MessageQueue.Internal.MessageBusInfo, false, true).BootstrapHandler(ctx, wg, startupTimer, dic) {
+		return
+	}
+	if !handlers.NewMessagingBootstrap(configuration.MessageQueue.External.MessageBusInfo, true, true).BootstrapHandler(ctx, wg, startupTimer, dic) {
+		return
+	}
```

Comment:
See other comments here and on bootstrap PR about using existing handler and creating new handler for External MQTT which avoids the need to put this outside the  bootstrap.Run()